src_rakuraku_pyqt5/input_form.py

```diff
@@ -518,7 +519,7 @@ def checkDiffMessage(self,exp=None):
             exp = "Yes を押すとこのページの変更が破棄されます。" 
 
         flag = self.checkDiffExist()
-        reply = QMessageBox.Yes
+        # reply = QMessageBox.Yes
```

Comment:
```suggestion
        reply = QMessageBox.Yes
```
もし、self.checkDiffExist()がFalseを返す、値の変更がないときは、reference before assignment のerrorが出てしまうので、#は外せない。